src/pybind/mgr/restful/module.py

```diff
@@ -244,18 +245,43 @@ def _serve(self):
             separators=(',', ': '),
         )
 
-        cert = self.get_config_json("cert") or '/etc/ceph/ceph-mgr-restful.crt'
-        pkey = self.get_config_json("pkey") or '/etc/ceph/ceph-mgr-restful.key'
+        server_addr = self.get_config('server_addr') or '127.0.0.1'
+        server_port = int(self.get_config('server_port') or '8003')
+        self.log.info('server_addr: %s server_port: %d',
+                      server_addr, server_port)
+
+        cert_fname = None
+        pkey_fname = None
+
+        cert = self.get_config("cert")
```

Comment:
This should be get_config_json, not get_config. get_config returns the cluster config as in /config/cluster, not the config-keys. (the same should go for other get_config calls).